libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -8,17 +8,31 @@
       <div class="reading-list-item--details--author">
         {{ b.authors.join(',') }}
       </div>
+      <div *ngIf="b.finished" class="reading-list-item--details--author" data-testing="toggle-finish-date">
+        Finish reading at {{ b.finishedDate }}
```

Comment:
Change it to: `Finished reading on {{ b.finishedDate }}`

Reply:
Done